lib/splitEvery/index.js

```diff
@@ -0,0 +1,9 @@
+import _curry2 from '../_internal/_curry2'
+
+export default _curry2(function splitEvery(num, arr) {
+  var res = []
```

Comment:
Mutable. Should we always use functional approach, or speed is in priority?

Reply:
We need a balance between them

Reply:
Why is this mutable?